cmd/janus/main.go

```diff
@@ -65,7 +65,7 @@ func main() {
 	defer accessor.Close()
 	defer statsdClient.Close()
 
-	statsClient := stats.NewStatsClient(statsdClient)
+	statsClient := stats.NewStatsClient(statsdClient, globalConfig.Statsd.DSN == "")
```

Comment:
I want Live and Muted clients to produce the same metrics, so I generalised incrementer and time tracker.